draft-ietf-ppm-dap.md

```diff
@@ -2382,12 +2381,13 @@ Next, the Helper checks the continuation step indicated by the request. If the
 `step` value is one greater than the job's current step, then the Helper
 proceeds.
 
-If it is equal to the current step (e.g., the Leader has resent the previous
-request), then the Helper MAY attempt to recover by sending the same response as
-it did for the previous `AggregationJobContinueReq`, without performing any
-additional work on the aggregation job. In this case it SHOULD verify that the
-contents of the `AggregationJobContinueReq` are identical to the previous
-message (see {{aggregation-step-skew-recovery}}).
+The Helper may receive multiple copies of a continuation request for a given
+step. The Helper MAY attempt to recover by sending the same response as it did
+for the previous `AggregationJobContinueReq`, without performing any additional
+work on the aggregation job. It is illegal to rewind or reset the state of an
+aggregation job, so in this case it MUST verify that the contents of the
+`AggregationJobContinueReq` are identical to the previous message (see
+{{aggregation-step-skew-recovery}}).
```

Comment:
Doing step skew recovery at all remains a MAY (i.e. the helper could just barf 400 Bad Request if it gets duplicate continue requests), but if you *are* doing recovery, then checking that the continue req for the given step hasn't changed is now a MUST. I believe this is what ekr meant by:

> I also wonder why checking that the parameters is the same is a MUST in 4.6.2.1 but not 4.6.3.4.